src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataManager.java

```diff
@@ -346,6 +346,8 @@ public static ArrayList parseHistoricalFile(String filename) throws IOException,
       // Read next line before looping again
       line = reader.readLine();
     }
+    reader.close();
```

Comment:
Should we even keep this old EBD to file code? I think it should probably just be removed to reduce confusion.

Reply:
Agreed. but I never did have the chance to test strings.... I got nervous when I had to reformat the Flexy.